app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -1006,12 +1006,16 @@ class MultipleDisposalsTriageController @Inject() (
                         val amendReturnOriginalTaxYear =
                           state.map(_._1.amendReturnData.map(_.originalReturn.completeReturn.taxYear)).toOption.flatten
                         taxYear match {
+                          case None if isAmendReturn(state) =>
```

Comment:
it'd be good to get a unit test for this new case and the other new case in the other controller